whole-body-controller-core/robot-arm-two/build.gradle.kts

```diff
@@ -13,10 +13,10 @@ ihmc {
 }
 
 mainDependencies {
-   api("us.ihmc:simulation-construction-set:0.22.10")
    api("us.ihmc:euclid:0.19.1")
    api("us.ihmc:simulation-construction-set-tools:0.14.0-230207")
    api("us.ihmc:ihmc-simulation-toolkit:0.14.0-230207")
```

Comment:
See if that dependency is still necessary

Reply:
Removed